postgres/macros.jinja

```diff
@@ -12,7 +12,9 @@
 
 {%- macro format_state(name, state, kwarg) %}
 
-  {%- do kwarg.update({'name': name}) %}
+  {%- if 'name' not in kwarg %}
+    {%- do kwarg.update({'name': name}) %}
+  {%- endif %}
```

Comment:
This is just awesome. Would mind to update `pillar.example` file for the formula users to see they now able to provide unique IDs for PG entities?

Reply:
You can't set state id via pillar. So no need for `pillar.example` update. This feature is for formula developers only.
IMHO Allowing user setting state ids is asking for trouble.

Reply:
@t0fik No-no, it is not about setting state IDs. There was PR #155 that intended to add capability to override PG entity attributes (including names) depending on some external data. But it wasn't merged. With your changes, it is also achievable.